docusaurus/docs/quickstarts/network/hosted.md

```diff
@@ -140,29 +138,78 @@ ubuntu@ip-10-0-0-1:~$ sudo systemctl -q status ziti-router --lines=0 --no-pager
              └─2385 /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ziti-bin/ziti-v0.22.11/ziti-router run /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ip-10…
 ```
 
+Inspect the servers' listening ports.
+
+```bash
+$ sudo ss -lntp | grep ziti
+LISTEN 0      4096               *:8440             *:*    users:(("ziti-controller",pid=5240,fd=10))
+LISTEN 0      4096               *:8441             *:*    users:(("ziti-controller",pid=5240,fd=9)) 
+LISTEN 0      4096               *:8442             *:*    users:(("ziti-router",pid=5272,fd=9))     
+LISTEN 0      4096               *:10080            *:*    users:(("ziti-router",pid=5272,fd=8))     
+```
+
+What are these server ports listening for?
+
+- `8440/tcp`: the Edge Controller listening for API requests. We specified this port as env var `ZITI_EDGE_CONTROLLER_PORT`. You need a firewall exception for this port to allow Ziti identities to obtain an Edge session.

```

Comment:
All of this was already lightly discussed above and feels redundant here. it also should just be generically documented anywhere else so that any/every quickstart can reference this same information. i think you should just remove this stuff and either figure out a way to generically document this, or just file an issue to do that work. as is, it'll have to be duplicated on any/every quickstart

Reply:
removed